index.src.html

```diff
@@ -731,7 +731,7 @@ <h4 id="interfaces-credential-types-passwordcredential"><code>PasswordCredential
       required USVString password;
     };
 
-    typedef (FormData or URLSearchParams) CredentialBodyType;
+    typedef (FormData or URLSearchParams or object) CredentialBodyType;
```

Comment:
> Here I hoped to make |object| point to https://heycam.github.io/webidl/#idl-object.

`object` is super generic. We might as well just drop the `typedef`, use `any` as the attribute's type, and do type checking explicitly inside the algorithm.

> I spotted that USVString has a link to the same spec, but could not find any definition causing that in the `<pre class="anchors">` section. How is this done for USVString?

Bikeshed relies on a backend service called "Shepherd", which parses specifications and indexes the definitions they export. In your Bikeshed directory, you'll see `./bikeshed/spec-data/anchors.json`, which has a full list of all the known terms.

This part happens to be in an `idl` block, though, which makes things a little more complicated. I'm not entirely sure whether the IDL parser integrated into Bikeshed uses the same set of anchors or not.

> Should I just add a "spec:" section for webidl?

No, if we can't get it working, the right thing to do would be to file a bug against Bikeshed and just deal with the missing link for the moment.